File: 프로그래머스/lv1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        int[] mon = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30};
        String[] day = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
        int num = 0;
        for(int i = 0; i < a; i++){
            num += mon[i];
        }
        num += b;
        num = num % 7;
        String answer = day[num];
        
        return answer;
    }
}

File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    answer = ''
    month = [31,29,31,30,31,30,31,31,30,31,30,31]
    day = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
    cnt = 0
    for i in range(a-1):
        cnt += month[i]
    cnt += b
    answer = day[(cnt+4)%7]
    return answer